archway/chain.json

```diff
@@ -258,7 +256,7 @@
       "version": "v7.4.0"
     },
     "cosmwasm": {
-      "version": "v0.45.0",
+      "version": "archway-wasmd v0.46.0-archway",
```

Comment:
repo: https://github.com/archway-network/archway-wasmd
version: v0.46.0
tag: v0.46.0-archway
(you can keep enabled)

will fail checks if not to following this format